components/skeleton/skeleton-group-mixin.js

```diff
@@ -0,0 +1,19 @@
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
+import { SubscriberRegistryController } from '../../controllers/subscriber/subscriberControllers.js';
+
+export const SkeletonGroupMixin = dedupeMixin(superclass => class extends RtlMixin(superclass) {
+
+	constructor() {
+		super();
+
+		this._skeletonSubscribers = new SubscriberRegistryController(this, 'skeleton', {
+			updateSubscribers: this._checkSubscribersReadyToDisplay.bind(this)
+		});
+	}
+
+	_checkSubscribersReadyToDisplay(subscribers) {
+		const readyToDisplay = ![...subscribers.values()].some(subscriber => subscriber.skeleton);
```

Comment:
Would it work to keep track of the previous `readyToDisplay` value and only update the subscribers if it's changed? Just thinking of a case where there's like 100 items and they each become ready independently but the whole thing isn't ready until they all are.

Reply:
Oh yeah, that's a good optimization. I'll play with that.